.claude/skills/merge-gate/SKILL.md

```
@@ -140,9 +144,86 @@ patient identifiers, credentials, tokens, and other sensitive fields from
+gh pr comment <PR> --body "..."
+```
+
+If merge-gate is re-run on the same PR later (e.g. after the author
+pushed fixes), post a **new** comment rather than editing/deleting the
+previous one — the history of gate runs staying visible is the point.
+
+Use an outcome-specific template so a PR author or a merger who wasn't in
+this session gets enough context without opening the chat transcript:
+
+**`PASSED`:**
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)